src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -2059,6 +2060,10 @@ in src-script.js and main.js
 		background: var(--main-background-color);
 	}
 
+    .item-table>li>div {
+        max-width: unset !IMPORTANT;
```

Comment:
Why is this needed?

Reply:
On mobile, the name stretches the whole way and the description comes below it. If I maintain it using only 40vw it will crop only halfway throughout the screen.

Reply:
I see. Then in this case, please remove this rule and move the other into the media query for desktop (which is at [this line](https://github.com/rust-lang/rust/blob/master/src/librustdoc/html/static/css/rustdoc.css#L2117)).